app/src/main/java/com/xmartlabs/template/di/FragmentBuildersModule.kt

```diff
@@ -14,11 +14,11 @@ abstract class FragmentBuildersModule {
   abstract fun contributeLoginFragment(): LoginFragment
 
   @ContributesAndroidInjector
-  abstract fun contributeOnboardingFragment(): OnboardingFragment
+  abstract fun contributeRecyclerExampleFragment(): RecyclerExampleFragment
```

Comment:
why?

Reply:
Like above I deleted some classes that belong to the base project